docs/contributing/coding-guidelines/ros-nodes/launch-files.md

```diff
@@ -1,5 +1,47 @@
+
+  As can be seen from the content, the entire launch file is divided into several different modules, including _Vehicle_, _System_, _Map_, _Sensing_, _Localization_, _Perception_, _Planning_, _Control_, etc. By setting `true` or `false`, we can determine which modules to be loaded.
+
+- The `logging_simulator.launch.xml` is often used together with the recorded ROS bag to debug if the target module (e.g. _Sensing_, _Localization_ or _Perception_) functions normally.
+
+- The `planning_simulator.launch.xml` is based on the Planning Simulator tool, mainly used for testing/validation of _Planning_ module by simulating traffic rules, interactions with dynamic objects and control commands to the ego vehicle.
+
+In Autoware, the nodes of each module is organized based on the [architecture](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-architecture/#high-level-architecture-design). So you may find that we try to match [launch structure](https://github.com/autowarefoundation/autoware.universe/tree/main/launch) similar to the architecture (splitting of files, namespace).
+
+```txt
+                                                        autoware_launch
+                                                                |
+─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+|                       |                           |                       |                       |                       |
+tier4_vehicle_launch    tier4_localization_launch   tier4_system_launch tier4_sensing_launch    tier4_perception_launch    ...
+                                |
+─────────────────────────────────────────────────────────────────
+|               |                   |                           |
+pose_estimator twist_estimator    pose_twist_fusion_filter     ...
+                                        |
+                    ─────────────────────────────────────────
+                    |                   |                   |
+                    ekf_localizer    stop_filter        twist2accel
+```
+
+### Create or add a new package in Autoware
+
+If a newly created package has executable node, we expect sample launch file and configuration within the package, just like the recommended structure shown in previous [directory structure](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/) page.
+
+In order to automatically load the newly added package when starting Autoware, you may choose to call its launch file at certain place. For example, if using ICP instead of NDT as the pointcloud registration algorithm, you can modify the `tier4_localization_launch/launch/pose_estimator/pose_estimator.launch.xml` file to load the newly added ICP package.
+
+## Future updates
+
+How we organize configuration files.
```

Comment:
We can work on this once https://github.com/autowarefoundation/autoware_launch/issues/66 is closed

Reply:
> We can work on this once [autowarefoundation/autoware_launch#66](https://github.com/autowarefoundation/autoware_launch/issues/66) is closed

Thanks! After the issue is finished, I will make some update.